metrics-operator/api/v1alpha3/zz_generated.deepcopy.go

```diff
@@ -25,6 +25,87 @@ import (
 	"k8s.io/apimachinery/pkg/runtime"
 )
 
+// DeepCopyInto is an autogenerated deepcopy function, copying the receiver, writing into out. in must be non-nil.
```

Comment:
```suggestion
// DeepCopyInto is an autogenerated deepcopy function, copying the receiver, writing intothe location specified for the out field. in must be non-nil.
```

Reply:
This file is completely auto-generated, I basically cannot modify it anyhow :/